db/schema.rb

```diff
@@ -371,9 +338,6 @@
     t.index ["proceeding_id"], name: "index_chances_of_successes_on_proceeding_id"
   end
 
-  create_table "data_migrations", primary_key: "version", id: :string, force: :cascade do |t|
```

Comment:
Isn't this table needed for `rake db:migrate:with_data`?
If not, you'll need to delete it from the anonymise rules file

Reply:
Thanks Colin, I think that has crept in from rebasing. I will fix this.